app/notifications/recipients/repository_item_recipients.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class Recipients::RepositoryItemRecipients
+  def initialize(params)
+    @repository_row_id = params[:repository_row_id]
+  end
+
+  def recipients
+    repository_row = RepositoryRow.find(@repository_row_id)
+    repository_row&.repository&.team&.users || []
```

Comment:
These & are unnecessary, since .find will throw an error anyway, if the row is not found.